web/src/routes/users/+page.svelte

```diff
@@ -30,6 +30,24 @@
 		$fetching = false;
 	};
 
+	let deleteAllUsersDialog: HTMLDialogElement;
+	const confirmDeleteAllUsers = () => {
+		deleteAllUsersDialog.showModal();
+	};
+	const deleteAllUsers = async () => {
+		$fetching = true;
+		const response = await fetch(API.ADMIN_USER_DELETE_ALL, {
+			method: "DELETE",
+		});
+
+		if (response.ok) {
+			location.reload()

```

Comment:
Not a very "react"ive way to do things but I see the overhead in making it so. Probably not worth it.